src/style/resume-simple/main.scss

```diff
@@ -0,0 +1,212 @@
+@import "bulma/bulma.sass";
+@import "../icons";
```

Comment:
do we need icons in for simple resume?

Reply:
no if we're following this structure example, at first ,i didn't mean to make it completely identical to this one:

![Screenshot from 2025-04-30 11-16-44](https://github.com/user-attachments/assets/32190e80-173d-4316-a4ed-10d5c9abe0f6)